xla/debug_options_flags.cc

```diff
@@ -1024,6 +1025,16 @@ void MakeDebugOptionsFlags(std::vector<tsl::Flag>* flag_list,
       bool_setter_for(&DebugOptions::set_xla_gpu_exclude_nondeterministic_ops),
       debug_options->xla_gpu_exclude_nondeterministic_ops(),
       "Excludes non-deterministic ops from compiled executables."));
+
+  flag_list->push_back(tsl::Flag(
+      "xla_gpu_disable_multi_threading_per_device",
+      bool_setter_for(
+          &DebugOptions::set_xla_gpu_disable_multi_threading_per_device),
+      debug_options->xla_gpu_disable_multi_threading_per_device(),
+      "If set to true, XLA will assume that there is no multi-threading "
```

Comment:
Is it possible to add larger motivation to PR description?

Reply:
Thanks, I added some description for this PR.